### PR TITLE
Update final report

### PR DESCRIPTION
**Pull request by jpabadir**

### CODE REVIEW

File: documentation/final-deliverables/pdf-report.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ FR23. To "unmatch" Mentees and Mentors, the Mentee can communicate with the Ment
 
 <br>
 
-## Limitations of MentorMatch (and future improvements)
+## Limitations of MentorMatch, and future improvements needed
```
Comment (katmartel) [inline documentation/final-deliverables/pdf-report.md]: This is a little nit-picky, but just for the sake of consistency 
 ```suggestion
## Limitations of MentorMatch, and Future Improvements Needed
```

File: documentation/final-deliverables/step-by-step-guide.md [hunk 2]
```diff
@@ -14,76 +12,63 @@ Install Node:
...
+3. Run `npm start`. This will start the MentorMatch app! You should see the homepage of MentorMatch open in your browser.
 4. Your front-end setup is done!
 
 ### Back-end
 
-1. In your terminal, navigate to pace-c/src/back-end. Assuming you're still in the front-end folder, you need to run `cd ../` to go up a level before running `cd pace-c/src/front-end` to access the back-end folder.
-2. Run npm install. This will install all required dependencies.
-3. Run nodemon `src/app.js`. Nodemon is a small package that helps us develop faster. All it does is detect when the file app.js has been modified, and stops and re-runs the server so we don't have to do it manually.
-4. Your back-end setup is done!
+Note that for the website to work properly, you need to run the backend at the same time as you run the frontend. Therefore, we suggested opening another terminal window or tab before running the instructions below.
```
Comment (katmartel) [inline documentation/final-deliverables/step-by-step-guide.md:27]: Should we also make a note about the 'secrets' file with the admin SDK configurations being required in order to run the back-end?

**Review by katmartel — APPROVED** [review]
Great work! :)

File: documentation/final-deliverables/step-by-step-guide.md (hunk 2 above)
Comment (jpabadir) [reply documentation/final-deliverables/step-by-step-guide.md:27]: I will do that, and the .env as well

**Review by wrc9817 — APPROVED** [review]
good job!

**Pull request merged**